cmd/sealer/cmd/image/load.go

```diff
@@ -40,28 +46,31 @@ func NewLoadCmd() *cobra.Command {
 		Long:    longNewLoadCmdDescription,
 		Example: exampleForLoadCmd,
 		Args:    cobra.NoArgs,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			engine, err := imageengine.NewImageEngine(options.EngineGlobalConfigurations{})
-			if err != nil {
-				return err
-			}
-
-			err = engine.Load(loadOpts)
-			if err == nil {
-				logrus.Infof("successfully load %s to image storage", loadOpts.Input)
-			}
-
-			return err
-		},
+		RunE:    runLoadCmd,
 	}
 	loadOpts = &options.LoadOptions{}
 	flags := loadCmd.Flags()
-	flags.StringVarP(&loadOpts.Input, "input", "i", "", "Load image from file")
+	flags.StringVarP(&loadOpts.Input, "input", "i", "", "Read from specified archive file")
 	flags.BoolVarP(&loadOpts.Quiet, "quiet", "q", false, "Suppress the output")
-	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "", "set temporary directory when load image. if not set, use system`s temporary directory")
+	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "", "Set temporary directory when load image. use system temporary directory is not (/var/tmp/)at present ")
```

Comment:
Set temporary directory when load image. use system temporary directory(/tmp/) if not presen.